src/custom/constants/index.ts

```diff
@@ -49,24 +49,28 @@ export const GP_SETTLEMENT_CONTRACT_ADDRESS: Partial<Record<number, string>> = {
   [ChainId.MAINNET]: GPv2Settlement[ChainId.MAINNET].address,
   [ChainId.RINKEBY]: GPv2Settlement[ChainId.RINKEBY].address,
   [ChainId.XDAI]: GPv2Settlement[ChainId.XDAI].address,
+  [ChainId.GOERLI]: GPv2Settlement[ChainId.RINKEBY].address, // TODO: update to GOERLI
```

Comment:
Probably Goerli should be added here https://github.com/cowprotocol/contracts/blob/main/networks.json

Reply:
Yep. To be sure we don't forget, I recommend to link this line in your gdoc regarding the changes and/or create an issue in the repo and link in the TODO